portal/models/reference.py

```diff
@@ -23,7 +23,7 @@ def organization(cls, organization_id):
     def patient(cls, patient_id):
         """Create a reference object from a known patient id"""
         instance = cls()
-        instance.patient_id = int(patient_id) if patient_id else None
+        instance.patient_id = int(patient_id)
```

Comment:
@pbugni I think one of the migrations Adam wrote for audits failed to account for something in procedures (I'm also seeing an error in running the migrations on my own instance)